store/migration/10001__init_schema.sql

```diff
@@ -59,6 +59,63 @@ VALUES
         ''
     );
 
+-- 3rd parties auth provider
+CREATE TABLE third_party_auth_provider
+(
+    id         INTEGER PRIMARY KEY AUTOINCREMENT,
+    row_status TEXT    NOT NULL CHECK (
+        row_status IN ('NORMAL', 'ARCHIVED')
+        )                       DEFAULT 'NORMAL',
+    creator_id INTEGER NOT NULL REFERENCES principal (id),
+    created_ts BIGINT  NOT NULL DEFAULT (strftime('%s', 'now')),
+    updater_id INTEGER NOT NULL REFERENCES principal (id),
+    updated_ts BIGINT  NOT NULL DEFAULT (strftime('%s', 'now')),
+    `type`     TEXT    NOT NULL CHECK (`type` LIKE 'bb.auth.provider.%'),
```

Comment:
For this table, I am more inclined to follow the pattern in vcs table

CREATE TABLE vcs (

`type` TEXT NOT NULL CHECK (`type` IN ('GITLAB_SELF_HOST')),

)

Reply:
Is there any reason?